types.go

```diff
@@ -106,6 +106,7 @@ const (
 
 	TypeTA       uint16 = 32768
 	TypeDLV      uint16 = 32769
+	TypeXPF      uint16 = 65422
```

Comment:
this can only be added once a code has been allocated. This is I believe in the private range; if you want this to work now in your code you'll need to fiddle with a PrivateRR.

Has a request been sent to IANA for a code allocation point?

Reply:
I'm not all too sure about the state of the proposal. It's something that PowerDNS seemed to have been trying to push through as they did the initial draft. 

I'm waiting to hear back from the original author(s) whether this was something that just got left out to dry, or if it's in limbo with IANA/IETF.

Reply:
Once you have a draft you can allocate a code. The process it relatively light weight nowadays, you need to fill out a form and give that to iana